src/software/geom/util_test.cpp

```diff
@@ -144,7 +144,7 @@ TEST(GeomUtilTest, test_proj_len)
 
 TEST(GeomUtilTest, test_contains_triangle_point)
 {
-    // this triangle lies in the first quatren of the field, we can rota
+    // this triangle lies in the point quatren of the field, we can rota
```

Comment:
It seems that refactoring went a bit too far?

Reply:
Fixed